### DISCUSSION TITLE
GitHub Support does not respond even after almost 15 days #2725005 (FiNiShYy)

### DISCUSSION THREAD

**Discussion by FiNiShYyy** [community/community#121011] [Other Feature Feedback, Questions, & Ideas]
I wrote a ticket to github about my account being flagged but they don't respond to me, this is my work account and I can't use it because it's flagged, I saw some posts about this same issue but apparently they only respond if you write A discussion. Please help
Iam using a secondary account because my primary account does not appear on anything

**Comment by FiNiShYy** [comment]
This is my primary account!!

**Comment by haroonrashid2210** [comment]
Same with my account

**Comment by ebndev** [comment]
Hi @FiNiShYyy,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion and any duplicate discussions.

We appreciate your understanding!
